diff --git a/dist/final-form.cjs.js b/dist/final-form.cjs.js
index 65fd607c183d7cd07e0d480477dc4355c2aa1884..0461c7cde0467cbabdd7b093d8016cf38eeab783 100644
--- a/dist/final-form.cjs.js
+++ b/dist/final-form.cjs.js
@@ -1,8 +1,20 @@
 'use strict';
 
-var _extends = require('@babel/runtime/helpers/extends');
 var _objectWithoutPropertiesLoose = require('@babel/runtime/helpers/objectWithoutPropertiesLoose');
 
+// https://github.com/final-form/final-form/issues/186#issuecomment-1735111004
+const _extends = function (target) {
+  for (var i = 1; i < arguments.length; i++) {
+    var source = arguments[i];
+    for (var key in source) {
+      if (Object.prototype.hasOwnProperty.call(source, key)) {
+        target[key] = source[key];
+      }
+    }
+  }
+  return target;
+}
+
 //      
 
 var charCodeOfDot = ".".charCodeAt(0);
diff --git a/dist/final-form.es.js b/dist/final-form.es.js
index e31d7f32c686fede8d70925fbd486679adc42b38..71b0f185e66ad9d56acb5bd26be097b5c558e7d3 100644
--- a/dist/final-form.es.js
+++ b/dist/final-form.es.js
@@ -1,6 +1,18 @@
-import _extends from '@babel/runtime/helpers/esm/extends';
 import _objectWithoutPropertiesLoose from '@babel/runtime/helpers/esm/objectWithoutPropertiesLoose';
 
+// https://github.com/final-form/final-form/issues/186#issuecomment-1735111004
+const _extends = function (target) {
+  for (var i = 1; i < arguments.length; i++) {
+    var source = arguments[i];
+    for (var key in source) {
+      if (Object.prototype.hasOwnProperty.call(source, key)) {
+        target[key] = source[key];
+      }
+    }
+  }
+  return target;
+}
+
 //      
 
 var charCodeOfDot = ".".charCodeAt(0);
